script/DeployLeveragedEthTestnet.s.sol

```diff
@@ -1,19 +1,157 @@
 // SPDX-License-Identifier: AGPL-3.0
 pragma solidity ^0.8.13;
 
+import "forge-std/console2.sol";
+import "forge-std/Test.sol";
+import "forge-std/StdStorage.sol";
+
 import {DeployLeveragedEth} from "./base/DeployLeveragedEth.sol";
 import {FixedPointMathLib} from "solmate/utils/FixedPointMathLib.sol";
 import {MockWETH} from "../test/mocks/MockWETH.sol";
 import {MockUSDC} from "../test/mocks/MockUSDC.sol";
-import {GoerliConstants as C} from "../src/lib/GoerliConstants.sol";
+import {Constants as C} from "../src/lib/Constants.sol";
+import {ERC20} from "solmate/tokens/ERC20.sol";
+import {sc4626} from "../src/sc4626.sol";
+import {scWETH} from "../src/steth/scWETH.sol";
+import {scUSDC} from "../src/steth/scUSDC.sol";
 
-contract DeployScript is DeployLeveragedEth {
-    function deployMockTokens() internal override {
-        weth = MockWETH(payable(C.WETH));
-        usdc = MockUSDC(C.USDC);
-    }
+contract DeployScript is DeployLeveragedEth, Test {
+    using stdStorage for StdStorage;
 
     function run() external {
-        deploy();
+        _deploy();
+        _fixtures();
+    }
+
+    function _fixtures() internal {
+        console2.log("\nexecuting steth fixtures");
+
+        _fund();
+
+        _deposit(weth, scWeth, alice, address(scWeth), C.INITIAL_WETH_DEPOSIT);
+        _deposit(weth, scWeth, bob, address(scWeth), C.INITIAL_WETH_DEPOSIT);
+        _deposit(usdc, scUsdc, alice, address(scUsdc), C.INITIAL_USDC_DEPOSIT);
+        _deposit(usdc, scUsdc, bob, address(scUsdc), C.INITIAL_USDC_DEPOSIT);
+
+        _rebalance(scWeth);
+
+        // double rebalance passes
+        _rebalance(scUsdc);
+        _rebalance(scUsdc);
+
+        _redeem(scWeth, alice);
+        _redeem(scWeth, bob);
+
+        _redeem(scUsdc, alice);
+        _redeem(scUsdc, bob);
+
+        _deposit(weth, scWeth, alice, address(scWeth), C.INITIAL_WETH_DEPOSIT);
+        _deposit(weth, scWeth, bob, address(scWeth), C.INITIAL_WETH_DEPOSIT);
+        _deposit(usdc, scUsdc, alice, address(scUsdc), C.INITIAL_USDC_DEPOSIT);
+        _deposit(usdc, scUsdc, bob, address(scUsdc), C.INITIAL_USDC_DEPOSIT);
+
+        _profit(); // create scUsdc profit scenario
+
+        _divergeLTV(scWeth);
+        _divergeLTV(scUsdc);
+    }
+
+    function _fund() internal {
+        console2.log("funding");
+
+        // Dole out ETH
+        deal(alice, C.INITIAL_WETH_FUNDING);
+        deal(bob, C.INITIAL_WETH_FUNDING);
+        deal(keeper, C.INITIAL_WETH_FUNDING);
+
+        // Dole out WETH
+        deal(address(weth), C.INITIAL_WETH_FUNDING * 2);
+        deal(address(weth), alice, C.INITIAL_WETH_FUNDING);
+        deal(address(weth), bob, C.INITIAL_WETH_FUNDING);
+
+        // Dole out USDC
+        deal(address(usdc), alice, C.INITIAL_USDC_FUNDING);
+        deal(address(usdc), bob, C.INITIAL_USDC_FUNDING);
+    }
+
+    function _deposit(ERC20 asset, sc4626 vault, address from, address to, uint256 amount) internal {
```

Comment:
add `_` prefix to the function parameters for consistency here and to the other functions below as well